### PR TITLE
chore: add JDNConvertibleCalendarDateAdapter and JDNConvertibleCalendar to monorepo

### PR DESCRIPTION
**Pull request by subotic**
resolves DEV-1915
resolves DEV-1916

### CODE REVIEW

**Review by mdelez — APPROVED** [review]
Before merging, it would be nice to explain to us the structure of the files and all of the tsconfig files

File: libs/jdnconvertiblecalendardateadapter/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "Angular Material Datepicker",
+    "DateAdapter",
+    "calendar",
+    "conversion"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dhlab-basel/JDNConvertibleCalendarDateAdapter.git"
+  },
+  "author": "Tobias Schweizer <t.schweizer@unibas.ch>, DHLab University of Basel (http://www.dhlab.unibas.ch)",
```
[the remark this answers is not in the document]
Comment (subotic) [reply libs/jdnconvertiblecalendardateadapter/package.json]: yep, done

File: libs/jdnconvertiblecalendardateadapter/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "description": "Implementation of Angular Material DateAdapter for JDNConvertibleCalendar",
+  "keywords": [
+    "Angular Material Datepicker",
+    "DateAdapter",
+    "calendar",
+    "conversion"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dhlab-basel/JDNConvertibleCalendarDateAdapter.git"
```
[the remark this answers is not in the document]
Comment (subotic) [reply libs/jdnconvertiblecalendardateadapter/package.json]: nope, removed

File: apps/dsp-app/src/main.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { environment } from './environments/environment';
 import { APP_CONFIG } from './app/main/declarations/dsp-api-tokens';
 
 function configListener() {
+    // just adding a comment
```
[the remark this answers is not in the document]
Comment (subotic) [reply apps/dsp-app/src/main.ts]: removed

**Pull request merged**